methods/main_methods.py

```diff
@@ -411,26 +411,32 @@ def show_distributions_together(labels, data):
         cnt += 1
         ax_r.grid()
         ax_r.set_title(PROPS[i])
+
+        colors_used = []
         for index, j in enumerate(grain_labels): # plots mask and distribution per class
             color_step = int((index + 1) * MAX_PIXEL / len(grain_labels))
+            colors_used.append(MAX_PIXEL - color_step)
             mask = np.ma.masked_where(labels!=j, MAX_PIXEL * np.ones(labels.shape) - color_step)
             ax_l.imshow(mask, alpha=0.8, cmap=CMAP, aspect='auto', vmin=0, vmax=MAX_PIXEL)
 
             ax_r.hist(data[:,:,i][labels == j], 50, alpha=0.8, density=True, color=CMAP(MAX_PIXEL - color_step))
 
     pyplot.tight_layout()
     pyplot.show()
+    return colors_used
```

Comment:
The colors are preserved to match the colors in show_distributions_separately method